frontend/public/redux.js

```diff
@@ -1,12 +1,28 @@
 import { applyMiddleware, combineReducers, createStore } from 'redux';
 import { reducer as formReducer } from 'redux-form';
-import thunk from 'redux-thunk';
 
 import { featureReducer, featureReducerName } from './features';
 import { monitoringReducer, monitoringReducerName } from './monitoring';
 import k8sReducers from './module/k8s/k8s-reducers';
 import UIReducers from './ui/ui-reducers';
 
+/**
```

Comment:
Curious if it would be helpful to make it a separate file here, `redux-thunk.js` just to make this really really obvious (should anyone care).